raft/util_test.go

```diff
@@ -64,10 +56,8 @@ func TestLimitSize(t *testing.T) {
 		{uint64(ents[0].Size() + ents[1].Size() + ents[2].Size()), []pb.Entry{{Index: 4, Term: 4}, {Index: 5, Term: 5}, {Index: 6, Term: 6}}},
 	}
 
-	for i, tt := range tests {
-		if !reflect.DeepEqual(limitSize(ents, tt.maxsize), tt.wentries) {
-			t.Errorf("#%d: entries = %v, want %v", i, limitSize(ents, tt.maxsize), tt.wentries)
-		}
+	for _, tt := range tests {
+		require.Equal(t, limitSize(ents, tt.maxsize), tt.wentries)
```

Comment:
Minor comment: Usually the first value is expected value, and the second one is actual one. 
```suggestion
		require.Equal(t, tt.wentries, limitSize(ents, tt.maxsize))
```